libs/frontend/feature-build/src/components/StyleRow.vue

```diff
@@ -36,15 +36,11 @@
     <div class="label">
       {{ propertyText }}
     </div>
-    <div class="item">
-      <div class="edit-item" :class="{ ['edit-item-inherited']: inheritedFrom }">
-        <div class="value-preview" :class="{ error }">
-          {{ model.value }}
-        </div>
-        <Edit class="edit-icon" @click="edit" />
-      </div>
-      <Minus v-if="!inheritedFrom" class="item-delete" @click.stop="removeStyle" />
+    <div class="value-preview" :class="{ error, ['value-inherited']: inheritedFrom }">
```

Comment:
I tried to simplify layout and style in a few right menu components, while fixing some UI bugs.